pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -110,6 +110,9 @@ func createDefaultSvc(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Com
 				},
 				Protocol: container.Ports[0].Protocol,
 			}
+			if servicePort.Name == "" {
```

Comment:
it is recommended to use

`len(string)==0` instead `string == ""` to check if string is empty.

this comment is not mandatory though